src/main/java/frc/utils/controllers/keyboard/KeyboardController.java

```diff
@@ -0,0 +1,150 @@
+package frc.utils.controllers.keyboard;
+
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.RobotManager;
+import frc.robot.constants.IPs;
+import frc.utils.CMDHandler;
+import frc.utils.dashboard.LoggedTableBoolean;
+
+import java.nio.file.Path;
+
+
+/*
+ * Inspired from Trigon code.
+ *
+ * @author Yoni Kiriaty, Trigon
```

Comment:
credit yishai levy, amit sucher, and adrian from bet shean

Reply:
trigon is good enough credit